src/gui/optionsdlg.cpp

```diff
@@ -77,13 +77,14 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     setModal(true);
 
     // Icons
-    m_ui->tabSelection->item(TAB_UI)->setIcon(GuiIconProvider::instance()->getIcon("preferences-desktop"));
+    m_ui->tabSelection->item(TAB_APPEARANCE)->setIcon(GuiIconProvider::instance()->getIcon("preferences-desktop"));
+    m_ui->tabSelection->item(TAB_BEHAVIOUR)->setIcon(GuiIconProvider::instance()->getIcon("gear"));
```

Comment:
There is no `preferences-desktop-theme.png` in the existing codebase. I didn't change any icon names - just moved around the ones that were already being used since we didn't have an icon for the Appearance tab.

I will revert them all (and not have an icon for Appearance) and you can do whatever you want to.